src/sql/base/browser/ui/infoButton/infoButton.ts

```diff
@@ -3,9 +3,12 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import 'vs/css!./infoButton';
 import { Button as sqlButton } from 'sql/base/browser/ui/button/button';
-import { IButtonOptions } from 'vs/base/browser/ui/button/button';
+import { addDisposableListener, EventType } from 'vs/base/browser/dom';
+import { IButtonOptions, IButtonStyles } from 'vs/base/browser/ui/button/button';
+import { Color } from 'vs/base/common/color';
+
+import 'vs/css!./infoButton';
```

Comment:
nit: style imports usually go at the top - would be good to keep that convention to more quickly know whether a component imports styles

Reply:
I wishthe organize import command can be customized